package.json

```diff
@@ -20,6 +20,11 @@
     "test": "jest",
     "test:watch": "jest --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint"
+    }
+  },
```

Comment:
Great!

Reply:
nit: Can we add `prettify` command as well? So prettier first and lint sencond.
```
"husky": {
    "hooks": {
      "pre-commit": ["yarn prettify", "yarn lint"]
    }
  },
```

Reply:
Sure thing - will add!